index.html

```diff
@@ -0,0 +1,200 @@
+                                <h2 class="program-list-item-container__title">
+                                    DM Exhibition
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Have the time to share your thoughts and opinions with our experts for each topic.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="far fa-comments"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    Forum
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Listen to speakers from various countries about the messeges of sharing and opening.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-praying-hands"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    Workshop
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Try creating your own work using open source license rather than just watching.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-project-diagram"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    DMC Ignite
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Get opporunites to network with DMC affiliates around the world, also after the summit.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">SEE THE WHOLE PROGRAM</button>
+                </div>
```

Comment:
- [ ] The `See the whole program` is simply a text in the original design not a button with a red background, kindly fix it.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128995230-22e40d54-9d20-4be2-8eb4-a06dce403ea9.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128995315-6de0c7c7-0e59-4eda-a6f6-dbc36d22e2e4.png' height='250px'> |